src/System.IO.FileSystem/tests/FileStream/ctor_str_fm.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.IO.Tests
 {
-    public class FileStream_ctor_str_fm : RemoteExecutorTestBase
+    public class FileStream_ctor_str_fm : RemoteFileSystemTest
```

Comment:
Because FileStream_ctor_str_fm_fa derives from FileStream_ctor_str_fm, FileStream_ctor_str_fm_fa_fs derives from FileStream_ctor_str_fm_fa,  FileStream_ctor_str_fm_fa_fs needs RemoteInvoke, and classes in .NET can have only one base class.